### PR TITLE
Create article-template.md

### PR DESCRIPTION
**Pull request by PhoenixGreen**
CT Article Template. Use this to get your article started.

### CODE REVIEW

File: demo-article.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+| Author | phoenixgreen |
+| :---- | :---- |
+| Title | Article Demo |
```
Comment (PhoenixGreen) [inline demo-article.md:4]: Can you think of a better title?

File: demo-article.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+| Twitter | post link… |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - What the article is about, and what is it aiming to prove
+
+Pre-discussion, research and questions
```
Comment (PhoenixGreen) [inline demo-article.md:19]: What about ... ?

File: demo-article.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+# Article / Title
+My article text goes here …
+
+The process of building a Decred block is actually pretty interesting and is the area that gives the project its security and efficiency.
```
Comment (PhoenixGreen) [inline demo-article.md:24]: Spelling error on ...

File: demo-article.md (hunk 1 above)
Comment (PhoenixGreen) [reply demo-article.md:4]: ...

**Comment by gregpent** [comment]
Great work 👏

**Pull request merged**